### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by benjub**
* moved pwuni up (by around 400 theorems) to a more appropriate subsection.
* uniexr and pwexr helped minimize several proofs and may have reduced axiom usage though I did not check.  The primary reason for separating them from the biconditional versions is that they do not require ax-un/ax-pow.  I did not add new credit (same credit as for the biconditional versions).
* added abnexg which generalizes abnex.  Edited some comments among abnex*, abrexex*; proved abrexex(2) from abrexex(2)g to shorten them.
* mathbox: Moore collections
* the rest is due to rewrapping of badly indented proofs
* I get a ` ?Warning: No page number after [<author>] bib ref in "nosepssdm". ` (@sctfn)

### CODE REVIEW

**Comment by sctfn** [comment]
Try a merge now - I missed a period.

**Comment by benjub** [comment]
Letting @jkingdon know about the characterization of stable classes.
Apparently, the double complement of a set need not even be a set in CZF, see https://mathoverflow.net/questions/296881/is-the-existence-of-double-complement-of-a-set-provable-in-intuitionistic-zf

**Comment by jkingdon** [comment]
> Letting @jkingdon know about the characterization of stable classes.

Ooh nice, thanks. Nice to get a biconditional out of that one. And cool to see that you are looking into CZF and some of the related topics (there's plenty of stuff in areas like that which I'm not aware of).

I'm not sure where I got the original wording of that comment. I suppose I was just going through set.mm and seeing where the proof relied on something iset.mm doesn't have. At least, I didn't cite any external sources and I don't remember consulting any (not that I likely would remember, because I don't really remember anything about this theorem at all).

In my next pull request I'll add the following to mmil.html (this work apparently predates the time when I was updating the missing theorems list more routinely):

```
<TR>
  <TD>ddif</TD>
  <TD>~ ddifnel , ~ ddifss</TD>
</TR>
```

**Review by jkingdon — APPROVED** [review]

**Comment by benjub** [comment]
Can anyone point me to some good documentation on merging ? Every time a conflict appears, it takes me hours to merge manually (even though I do a very automatic task of merging everything).

What I currently do: when I obtain
```
(misc)$ git merge develop 
Auto-merging set.mm
CONFLICT (content): Merge conflict in set.mm
Auto-merging discouraged
Automatic merge failed; fix conflicts and then commit the result.
```
I type
```
(misc|MERGING)$ git mergetool
Merging:
set.mm

Normal merge conflict for 'set.mm':
  {local}: modified file
  {remote}: modified file
Couldn't set the locale: unsupported locale setting; falling back to 'C' locale

(meld:8472): Gtk-WARNING **: 14:33:17.758: Locale not supported by C library.
	Using the fallback 'C' locale.
```
which opens Meld, with three files (LOCAL, ..., REMOTE), with highlighted parts that I have to decide either to merge from the left or the right to the middle column (but I also have to push the middle column to the other side, to have three identical files, which should not be needed).  Actually, more than 90% of highlighted parts come from only one of the sides, so there are no conflicts at all. What doesn't git understand ?

**Comment by benjub** [comment]
Ok, merge #2367 instead of this one, which git does not let me fix...

**Comment by benjub** [comment]
I did a `push origin misc --force` which apparently worked, so closing #2367.
@sctfn : some of your proofs were not rewrapped, so that the standard script at https://github.com/metamath/set.mm/blob/develop/CONTRIBUTING.md modified your mathbox, which I think was one cause of some conflicts.

**Comment by sctfn** [comment]
Okay, thanks. To note, I’ll push unwrapped changes to my branch to move them between my various computers. You shouldn’t merge origin/sctfen in until I PR it.

**Comment by benjub** [comment]
Unless I misunderstand, I did not merge from your branch. The only merges I do are from `develop` and are in the sequence:
```
(develop)$ git pull upstream develop
(develop)$ git checkout misc
(misc)$ git merge develop
```
But I'm not git-fluent, so I may be misdoing some things.

**Comment by jkingdon** [comment]
> Actually, more than 90% of highlighted parts come from only one of the sides, so there are no conflicts at all.

If you want to keep using meld, sounds like you are looking for the "Changes | Merge all non-conflicting" menu item which should take care of the >90% described here with no need to click on each chunk (based on what I read on the internet and what I've seen in similar tools).

What I do, instead of mergetool/meld, is open the conflicted file in a text editor and look for the `<<<<<<<` markers as shown at the "Basic Merge Conflicts" section of https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging . This might be more error prone or hard to follow than Meld but it is one option.

**Review by jkingdon — APPROVED** [review]

**Comment by jkingdon** [comment]
I clicked "Squash and merge" on this pull request because "Create a merge commit" (which I think is what we have been doing historically) is disabled with "Not enabled for this repository". In this case "Rebase and merge" would require resolving conflicts for each commit, which seems very much not worth it. @david-a-wheeler did we change something or am I confused?

**Comment by sctfn** [comment]
Okay, I must have forgotten to rewrap before I sent it up. Sorry!

**Comment by jkingdon** [comment]
> Okay, I must have forgotten to rewrap before I sent it up. Sorry!

I think we've talked about automating a "must rewrap" check as part of the automated checks, although I suppose we'll get around to that when we get around to it.

**Comment by david-a-wheeler** [comment]
I didn't change any settings. You should be able to do a merge commit. I'll have to check...
